src/openklant/components/klantinteracties/openapi.yaml

```diff
@@ -3221,6 +3221,22 @@ components:
       required:
       - url
       - uuid
+    ExpandDigitaalAdres:
```

Comment:
I do see the expand show up in the response schema, but I do not see the query parameter for expand in the list endpoint, could you add this? I suspect it might be missing for other endpoints as well

![image](https://github.com/user-attachments/assets/487659cf-99e1-42f9-9e69-7b5363304a4e)